pkg/reconciler/autoscaling/kpa/kpa_test.go

```diff
@@ -282,6 +282,8 @@ func TestReconcile(t *testing.T) {
 	deciderKey := struct{}{}
 	retryAttempted := false
 
+	asConfigKey := struct{}{}
```

Comment:
this is only in the tests.. is it doing anything?

Reply:
yeah, in order to test the initial scale 0 case we need to set the AllowZeroInitialScale flag.

Reply:
Im probably missing something, but I can't see where this is actually used though.. it's just put in the WithContexts in the table rows, but nothing reads with this struct as a key? (the actual autoscaler config key is an entirely different struct)

Reply:
Yeah you are right 🤦‍♀️ i'll fix this